app/controllers/application_controller.rb

```diff
@@ -1,4 +1,3 @@
 class ApplicationController < ActionController::Base
-  include DeviseTokenAuth::Concerns::SetUserByToken
   protect_from_forgery with: :null_session
```

Comment:
Becarefull if you transfer the logic for Api un a new controller this should go in too.

Reply:
Nothing on this?